contracts/DelayedClaimable.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.4.4;
+import './Ownable.sol';
+import './Claimable.sol';
+
+/*
+ * DelayedClaimable
+ * Extension for the Claimable contract, where the ownership needs to be claimed before certain block number
+ */
+
+contract DelayedClaimable is Ownable, Claimable {
+  uint public claimBeforeBlock;
+
+  function setClaimBefore(uint _claimBeforeBlock) onlyOwner {
+    claimBeforeBlock = _claimBeforeBlock;
+  }
+
+  function claimOwnership() onlyPendingOwner {
+    if (block.number > claimBeforeBlock) throw;
```

Comment:
Just realized there's a big conceptual problem, haha. Sorry I missed this before.

The concept of "delayed claimable" means you have a delay before you can actually claim it. In this case, `claimBeforeBlock` is an expiration timer where the new owner can't claim it after that time has passed.

The logic should be reverse! i.e: `if (block.number < claimAfterBlock) throw;`